portal/src/texts/developer/TestGuide.md

```diff
@@ -7,17 +7,15 @@ path: /utvikler/tester
 
 Det er mange som er avhengige av at Jøkul virker, men vi har ikke ressurser til å teste manuelt for regresjon ved hver endring. Derfor trenger vi gode automatiske tester som øker tilliten til at Jøkul virker. Samtidig ønsker vi ikke å bruke tid på å skrive tester som ikke gir verdi, eller teste funksjonalitet som er godt testet andre steder i verdikjeden.
 
-Vi har tre nivåer med tester: Enhetstester, integrasjonstester og visuelle regresjonstester. De første to typene kjøres automatisk på bygg-serveren vår, mens de visuelle regresjonstestene kjøres manuelt i forkant av en ny utgivelse.
-
 ## Enhetstester
 
-> Enhetstestene finnes i `<MinKomponent>.test.ts(x)`
->
-> Disse testene skal sikre at nøkkelfunksjonaliteten til komponenten virker.
+Enhetstester er første nivå av tester, og det du trenger å forholde deg til når du skriver komponenter til Jøkul. De skal ligge sammen med komponenten med filnavn på formen `<MinKomponent>.test.tsx` eller `<minFunksjon>.test.ts`.
+
+> Enhetstestene skal sikre at nøkkelfunksjonaliteten til komponenten virker.
 
 Eksempelvis kan testene for en knapp sjekke at den blir rendret til DOM, og at den kaller riktig funksjon ved klikk. Så langt det er fornuftig bør man ha 100% branch coverage på komponenter, men bruk sunn fornuft. Skriver man støttefunksjoner bør disse også ha 100% branch coverage, og dekke alle edge-verdier.
 
-Kjerneteamet har ikke satt opp noen formelle testdekningskrav, men dette er noe vi ser på i forbindelse med pull requester. Vi bruker `Jest` som test runner og `React testing library` for å håntere React-komponenter. Testene skal ligge sammen med koden, og enhetstestene har postfikset `.test.ts(x)`.
+Designsystemteamet har ikke satt opp noen formelle testdekningskrav, men dette er noe vi ser på i forbindelse med pull requester. Vi bruker [Jest](https://jestjs.io/) som test runner og [React testing library](https://testing-library.com/docs/react-testing-library/intro) for å håntere React-komponenter. Testene skal ligge sammen med koden, og enhetstestene har postfikset `.test.ts(x)`.
```

Comment:
~~håntere~~  -> håndtere

Er postfiks et ord? Translate sier "suffiks", men ingen av de er akkurat dagligtale for meg så vet ikke hva som er best her. Kanskje "[...] og enhetstestene skal ha endelsen `.test.ts(x)`."?